### PR TITLE
disabling mfT notification and menu item after first operator usage

### LINKED ISSUE #120: mirrord for Teams -> Try it now menu item shouldn't show for users who have mirrord for Teams
See title

### PR DESCRIPTION
**Pull request by Razz4780**
Closes #120

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -208,16 +208,23 @@ export class MirrordExecution {
   env: Map<string, string>;
   patchedPath: string | null;
   envToUnset: undefined | string[];
+  usesOperator?: boolean;
```
Comment (meowjesty) [inline src/api.ts:211]:
```suggestion
  ¿usesOperator?: boolean;
```

I think we could use a little bit more spanish in our code.

**Review by meowjesty — APPROVED** [review]
:+1:

File: src/api.ts (hunk 1 above)
Comment (Razz4780) [reply src/api.ts:211]: what kind of SeñorScript this is

Comment (meowjesty) [reply src/api.ts:211]: :maracas:

**Pull request merged**